devservices/config.yml

```diff
@@ -27,6 +27,11 @@ services:
     ports:
       - 127.0.0.1:7899:7899
     command: [run, --config, /etc/relay]
+    healthcheck:
+      test: curl -f http://127.0.0.1:7899/api/relay/healthcheck/live/
```

Comment:
What does devservices do with this healthcheck? Should it check for liveness or for readiness?

Reply:
Should I check for both? I just want to know generally if relay is up and running and can serve traffic

Reply:
Serve traffic would be the `ready` check, `live` is for whether Relay needs to be restarted.

>  if relay is up and running and can serve traffic

Sounds like you want the `ready` check.

Reply:
Thanks. If the `ready` check succeeds, does that mean the `live` check should also pass?

Reply:
Yes, if ready passes, live will as well.

Reply:
On second look, it seems like in order for this healthcheck to pass with sentry, we'd need the sentry devserver to be up and running. This is not inline with what we want, at least right now so I'm going to change this back to `/live/`.